docs/quick-start.md

```diff
@@ -211,8 +181,53 @@ collections:
+
+### Enable Identity and Git Gateway
+Netlify's Identity and Git Gateway services allow you to manage CMS admin users for your site without requiring them to have GitHub accounts or commit access on your repo. From your site dashboard on Netlify:
+ 1. Go to **Settings > Identity**, and select **Enable Identity service**.
+ 2. Under **Registration preferences**, select **Open** or **Invite only**. In most cases, you'll want only invited users to access your CMS, but if you're just experimenting, you can leave it open for convenience.
+ 3. If you'd like to allow one-click login with services like Google and GitHub, check the boxes next to the services you'd like to use, under **External providers**.
+ 4. Scroll down to **Services > Git Gateway**, and click **Enable Git Gateway**. This will authenticate with GitHub and generate a GitHub API access token. In this case, we're leaving the **Roles** field blank, which means any logged in user may access the CMS. For information on changing this, check the [Netlify Identity documentation](https://www.netlify.com/docs/identity/).
+
+### Add the Netlify Identity Widget
+With the backend set to handle authentication, now you need a frontend interface to connect to it. The open source Netlify Identity Widget is a drop-in widget made for just this purpose. To include the widget in your site, you'll need to add the following script tag in two places:
+
+```html
+<script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+```
+
+You'll need to add this to the `<head>` of your CMS index page at `/admin/index.html`, as well as the `<head>` of your site's main index page. Depending on how your site generator is set up, this may mean you need to add it to the default template, or to a "partial" ir "include" template. If you can find where the site stylesheet is linked, that's probably the right place.
+
+When a user logs in with the Netlify Identity widget, they will be directed to the site homepage with an access token. In order to complete the login and get back to the CMS, we'll need to redirect the user back to the `/admin` path. To do this, add the following script before the closing `body` tag of your site's main index page:
+
+```html
+<script>
+  if (window.netlifyIdentity) {
+    window.netlifyIdentity.on("init", user => {
+      if (!user) {
+        window.netlifyIdentity.on("login", () => {
+          document.location.href = "/admin/";
+        });
+      }
+    });
+  }
+</script>
+```
+*Note this script require ES6 and will not work on IE11. For legacy browser support, you'll need to use a transpiler like [Babel](https://babeljs.io/).
```

Comment:
"...For legacy browser support, use function expressions (`function () {}`) in place of the arrow functions (`() => {}`), or use a transpiler like...".

Also remove prepended asterisk.

Reply:
What if I just convert the code to use function expressions instead? Or do you think it's better to use the arrow functions and include the note?

Reply:
I'd say use arrows and include a note; I think we should default to ES6/ES2015/whatever it is now in docs, since that's what we use in the code. Do you think it would be a good idea to include a link to another version of it in ES5, or is that not worth the risk of them not getting updated simultaneously in the future?